saleor/payment/gateways/braintree/tests/test_braintree.py

```diff
@@ -315,7 +315,7 @@ def sandbox_braintree_gateway_config(gateway_config):
 
 
 @pytest.mark.integration
-@pytest.mark.vcr(filter_headers=["authorization"])
+@pytest.mark.vcr(filter_headers=["authorization"], decode_compressed_response=True)
```

Comment:
VCR stopped decompressing gzip responses by default, thus we now tell explicitly to decompress them.

It's likely an incompatibility caused by using older YAML cassettes, it may no longer be an issue if the YAML files were to be regenerated (e.g., the `Accept` header would be different causing us to no longer receive compressed responses), however I'm not sure.